docs/application/tizen-studio/release-notes/2-4-release-notes.md

```diff
@@ -0,0 +1,86 @@
+-	Certificate Manager
+	-   Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+	-   IoT certification currently requires a user to manually download and select the certificate.
+-	Native UI Builder
+	-   If the expanded attribute in a multibutton entry component is set to false, + is displayed.
+-	Native Component Designer
+	-   The vector-type part is not supported. You cannot see the vector image and change the SVG file.
+	-   Playing sound is not supported on Windows&reg; or macOS.
+	-   The Component Designer crashes if an alias is selected as the source group of an added item.
+-	Emulator
+	-   To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for the Tizen Studio](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/prerequisites).
+	    -   If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows&reg;, you must set the Tizen emulator to run with your NVIDIA graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee ). For Windows&reg;, select *High Speed NVIDIA Processor* as *Preferred Graphics processor* in the NVIDIA control panel.
+	    -   On Ubuntu, if the graphics driver is out-of-date, your Ubuntu desktop session occasionally logs out while launching the Emulator Manager, or the emulator skin is displayed improperly. Verify the prerequisites and upgrade to the latest graphics driver.
+	-   On Ubuntu 14.04, a shortcut menu can sometimes appear transparent.
+	-   On Windows&reg;, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
+	-   On Windows&reg;, if an error with message "failed to allocate memory" occurs while executing the emulator, try the following:
+	    -   Close some other programs and try to launch the emulator again.
+	    -   If the RAM size is set to 768 or 1024 MB for the VM in the Emulator Manager, change it to 512 MB.
+	    -   Increase the user area of the virtual memory in the system to 3 GB by entering the *bcdedit /setincreaseuserva 3072* command on the console with administrator rights (only in Windows&reg; 7), and reboot.
+	-   If you use a MacBook Pro which has both Intel HD and NVIDIA&reg; GPUs, the emulator can unexpectedly terminate when you execute the emulator with OpenGL ES version set v1.1 & v2.0. Verify the emulator configuration in the Emulator Manager and on the general tab in the emulator configuration window, set OpenGL ES version to v2.0 & v3.0.
+	-   When you launch the Emulator Manager in the Tizen IDE, the Emulator Manager's shortcut image may not be displayed properly.
```

Comment:
When you launch the Emulator Manager in the Tizen IDE, the shortcut image of Emulator Manager may not be displayed properly.